github/Repository.py

```diff
@@ -621,6 +621,14 @@ def teams_url(self):
         return self._teams_url.value
 
     @property
+    def topics(self):
+        """
+        :type: list 
```

Comment:
Just curious, what does adding trailing space do?
30f25d1ea8e9fc93248b01adb04ca8ff418f8710

Reply:
I was just trying to kick travis.  It got hung up on building and "failed" the build. It was me just bumping it to get a successful build